Backports the fix for https://rustsec.org/advisories/RUSTSEC-2026-0097.html

'Upstream' fix can be seen in the 0.9.4 upgrade at crrev.com/c/7767186

diff --git a/src/rngs/adapter/reseeding.rs b/src/rngs/adapter/reseeding.rs
index ae3fcbb2f..d5ea98fea 100644
--- a/src/rngs/adapter/reseeding.rs
+++ b/src/rngs/adapter/reseeding.rs
@@ -243,15 +243,15 @@ where
     ) {
         #![allow(clippy::if_same_then_else)] // false positive
         if self.is_forked(global_fork_counter) {
-            info!("Fork detected, reseeding RNG");
+            // info!("Fork detected, reseeding RNG");
         } else {
-            trace!("Reseeding RNG (periodic reseed)");
+            // trace!("Reseeding RNG (periodic reseed)");
         }
 
         let num_bytes = results.as_ref().len() * size_of::<<R as BlockRngCore>::Item>();
 
         if let Err(e) = self.reseed() {
-            warn!("Reseeding RNG failed: {}", e);
+            // warn!("Reseeding RNG failed: {}", e);
             let _ = e;
         }
         self.fork_counter = global_fork_counter;
